package.json

```diff
@@ -26,22 +26,22 @@
     "test": "grunt test"
   },
   "dependencies": {
-    "showdown": "0.x.x"
+    "showdown": "1.0.2"
   },
   "devDependencies": {
-    "chai": "^1.10.0",
+    "chai": "^3.0.0",
```

Comment:
No of course not but I tend to try to stay updated and two major versions seems like a lot...